surveys/2023-annual-survey/questions.md

```diff
@@ -594,6 +594,19 @@ Aspects:
 >
 > This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
 
+### How do you download crates to build Rust projects?
+
+Type: matrix (optional)
+
+Aspects:
+- I use crates.io
+- I use a custom/local/company registry
+- I don't know
```

Comment:
```suggestion
- I use crates.io
- I use a custom/local/company registry
- I use a mirror
- I don't know
```

Suggested [here](https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/Rust.20Survey.202023/near/396412370) to add the option of users using a mirror

Reply:
Does "mirror" mean "a mirror of crates.io"? If yes, then possibly we could explicitly specify that. Otherwise I'm not sure what's the difference between a mirror and a custom registry.